raiden/raiden_service.py

```diff
@@ -166,14 +195,17 @@ def message_for_task(self, message, hashlock):
         Return:
             bool: True if a correspoding task is found, False otherwise.
         """
+        # allow multiple managers to register for the hashlock (used for exchanges)
+        found = False
+
         for asset_manager in self.managers_by_asset_address.values():
             task = asset_manager.transfermanager.transfertasks.get(hashlock)
 
             if task is not None:
                 task.on_response(message)
-                return True
+                found = True
```

Comment:
That hides the number of found tasks. Not sure if we need to know about it though. How about returning the count of found tasks instead?